src/main/fc/fc_core.c

```diff
@@ -462,6 +462,13 @@ void processRx(timeUs_t currentTimeUs)
 
     updateActivatedModes();
 
+#ifdef USE_DSHOT
```

Comment:
This seems to be a bit of an unintuitive place to put this. Is there any other task (thinking motor update) where this would be more appropriate (thinking of ease of finding it when looking for it).

Reply:
Mainly because it is a warning that a mode is activated. Functionally it has little to do with the motors other than what protocol is in use.

Reply:
Yeah, you are right, on closer look this seems to be a good place for it to be. The culprit here is the name of the function being `processRx`, when it really should be something like `updateState`, considering it does a couple of hundred lines of exactly that...

Reply:
Yes, I think the main thing I was looking for is it being after the call to `updateActivatedModes()`